app/helpers/application_helper.rb

```diff
@@ -41,6 +41,21 @@ def indent(w=10)
     "<span style='margin-left:#{w}px'>&nbsp;</span>".html_safe
   end
 
+  # Wrap content in tag if condition is true, else nil
+  # 1st arg is condition, which wraps content_tag
```

Comment:
In my view these functions are clear enough that they don't need comments.  Not a blocker for accepting the pull.